src/test/java/com/itacademy/cms/CategoryTest.java

```diff
@@ -0,0 +1,138 @@
+//package com.itacademy.cms;
```

Comment:
please provide valid working tests
do not commit commented code

Reply:
Add @Ignore for CategoryTest test-methods